web-ui/src/main/resources/ui/app/Scatter2D.js

```diff
@@ -20,135 +20,38 @@ define(function(require) {
   Scatter2D.prototype.parent = Visualization.prototype;
 
   Scatter2D.prototype._draw = function(pointsWithTypes) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 6.](https://www.codacy.com/app/frank_7/spawncamping-dds/file/2019424934/issues/source?bid=3046234&fileBranchId=3126677#l22)